VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/participants-panel/participants-panel.component.scss

```diff
@@ -0,0 +1,94 @@
+@import '~govuk-frontend/govuk/all';
+
+.panel-container {
+  background: govuk-colour('black');
```

Comment:
We should avoid fixing the position of the component here, rather have the judge waiting room dictate where to place the participants panel

Reply:
I created css class to wrap panel on the host page and removed positioning from the panel.